lib/suspenders/app_builder.rb

```diff
@@ -270,6 +270,10 @@ def setup_stylesheets
         'app/assets/stylesheets/application.css.scss'
     end
 
+    def setup_autoprefixer
+      copy_file 'browserslist', 'browserslist'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.